qCC/main.cpp

```diff
@@ -89,14 +90,11 @@ int main(int argc, char **argv)
 	bool commandLine = (argc > 1) && (argv[1][0] == '-');
 #endif
    
-	if ( !commandLine )
```

Comment:
Actually, I'm not sure to see what was the advantage of removing this test? Does it have an impact on the translation stuff?

Reply:
When starting cloud compare from the command line with a supplied -LANG argument it sets commandLine true, which prevents running the InitOpenGL call. That causes going into full screen mode to hard crash.
See #1387

Reply:
Ah yes, sorry. Can't we just add a specific test case against a single '-LANG' command? Or avoid using '-'?

Reply:
we can if we want. 
looking at the code executed in InitOpenGL didn't seem to have any real impact unless you later created a GL window or widget, I couldn't find any connection to OpenGL directly, instead it seems to be configuring QT to deal with OpenGL later.

I'm ok with whichever method you prefer. :)

Reply:
Yes, that's probably right. I'm just a little bit fearfull of changing something that might impact the command line mode (while not being 'absolutely required').

On the other hand, we'll hear about users soon enough 😈 . Let's go with this version!